region/brinstar/green.json

```diff
@@ -2534,13 +2544,28 @@
                   ]
                 },
                 {
-                  "name": "Etecoon Spikeway Wiggle",
+                  "name": "Etecoon Spikeway Aim Cancel Wiggle",
                   "notable": true,
                   "requires": [
                     "canTurnaroundAimCancel",
                     "canBePatient",
                     {"thornHits": 180}
                   ]
+                },
+                {
+                  "name": "Etecoon Spikeway Xray Wiggle (Left to Right)",
+                  "notable": true,
+                  "requires": [
+                    "canXrayWaitForIFrames",
+                    "canXrayTurnaround",
+                    {"thornHits": 2}
```

Comment:
looks like bobbob only took 1 thorn hit. Or is this as an added leniency? (if so, mention that in a devnote)

Reply:
I was doing the tunnel crawl start so it was an extra hit for the complexity.  But there is no reason to start wiggling that way.